ome/training-notebooks
# Cp re organise

Re-organise the CellProfiler folder so we can add more pipelines and notebooks

To test:
* Use outreach trainer-1 dataset 1996
* Check that the notebook still works

Can be tested at https://ome-lochy.openmicroscopy.org/jupyterhub-training
